S/SFML/build_tarballs.jl

```diff
@@ -0,0 +1,82 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder
+
+name = "SFML"
+version = v"2.5.1"
+
+# Collection of sources required to build SFML
+sources = [
+    GitSource(
+        "https://github.com/SFML/SFML.git",
+        "2f11710abc5aa478503a7ff3f9e654bd2078ebab",
+    ),
+    ArchiveSource(
+        "https://www.sfml-dev.org/files/SFML-2.5.1-linux-gcc-64-bit.tar.gz",
+        "34ad106e4592d2ec03245db5e8ad8fbf85c256d6ef9e337e8cf5c4345dc583dd",
+    ),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+# build SFML
+cd ${WORKSPACE}/srcdir
+
+if [[ "${target}" == *linux* ]]; then
+
+cd SFML-2.5.1/
+mv ./include $WORKSPACE/destdir/
+mv ./lib64 $WORKSPACE/destdir/
```

Comment:
Please don't hardcode `$WORKSPACE/destdir/`: I don't think the documentation recommends using it anywhere (and if it does, it's an error), that path may change in the future, and it is even much longer to type than `${prefix}`
```suggestion
mv ./include ./lib64 ${prefix}
```
Also, [this question](https://github.com/JuliaPackaging/Yggdrasil/pull/1722#discussion_r498364581) still stands